app/data/domain/cdoTaskList.js

```diff
@@ -91,13 +91,16 @@ class CdoTaskList {
   }
 
   get insuranceDetailsRecorded () {
-    const [insurance] = this._cdo.exemption.insurance
+    const completed =
+      this.cdoSummary.insuranceCompany !== undefined &&
+      CdoTaskList.dateStageComplete(this.cdoSummary.insuranceRenewal) &&
+      dateTodayOrInFuture(this.cdoSummary.insuranceRenewal) &&
+      this._cdo.exemption.insuranceDetailsRecorded !== null
```

Comment:
I'm sure it still works fine but a bit of inconsistency here. In these three 'completed' values you are checking the date 3 different ways:
1. Date !== null
2. Date instanceof Date
3. Date
Would be nice to have a consistent way of checking them all